### ISSUE TITLE
UI elements with low color contrast ratios

### ISSUE THREAD

**Issue by JakeH** [MelvorIdle/melvoridle.github.io#1420]
Hello!

UI elements with either light green or light blue backgrounds, with white foreground text (such as LEVEL, XP, and various buttons) are difficult to read due to their low contrast ratio. 

Neither color combination passes the Web Content Accessibility Guidelines for contrast. I would like to suggest either the background or foreground colors be changed to pass at least WCAG AA standards in order to offer better readability.

As it stands right now, it is a bit difficult for me to read the text on the screen, so implementing this change would allow me to better understand the content of the game much more easily. 

Thank you. I'm very much enjoying the game so far. The quality of everything is just so premium. 

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=30C78D
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=5CACE5
https://www.w3.org/WAI/standards-guidelines/wcag/

**Comment by Bian18** [comment]
Not gonna lie this seems incredibly pedantic. I clicked on your links and it's not even hard to read in the slightest...
Even WebAIM doesn't meet WCAG AAA standards, because they're clearly setting the bar too high.

**Comment by JakeH** [comment]
@Bian18 ,
> Not gonna lie this seems incredibly pedantic. 
Thank you for not lying. 

My monitor is calibrated using an external tool, and as it stands, the color contrast is difficult for me to read. Perhaps your display is set in a way which makes this easier for you to read. 

I submitted this as a suggestion. If it's rejected as being too pedantic by the devs, then that's that.

**Comment by MelvorIdle** [comment]
Thanks for the suggestion. We have noted this, but are closing all suggestions on GitHub to clean up as this issue tracker will not be used for Bugs only.
